crates/turbopack-resolve/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "turbopack-resolve"
+version = "0.1.0"
+description = "TBD"
```

Comment:
coincidentally we just discussed this (https://vercel.slack.com/archives/C03EWR7LGEN/p1709141958393789?thread_ts=1709077393.256749&cid=C03EWR7LGEN), mind leave a one liner for the descrription?